modules/test/playwright/helpers/ListTypeAdminApiHelper.ts

```diff
@@ -40,12 +40,14 @@ export class ListTypeAdminApiHelper {
 
 	async postListTypeEntry(
 		listTypeDefinitionExternalReferenceCode: string,
-		listTypeEntryName: string
+		EnglishListTypeEntryName: string,
```

Comment:
can you change this prop to start with a lowercase letter? like `englishListTypeEntryName`